docs/en/observability/apm/getting-started-apm-server/index.asciidoc

```diff
@@ -0,0 +1,153 @@
+
+[cols="1,3"]
+|===
+| *Pros*
+a| Conveniently manage one, some, or many different
+integrations from one central {fleet} UI.
+
+| *Supported outputs*
+a| * {es}
+* {ess}
+
+| *Required{nbsp}components*
+a| * APM agents
+* APM Server
+* {agent}
+* Fleet Server
+* {stack}
+
+| *Configuration method*
+| {kib} UI
+|===
+
+// [float]
+// [[apm-setup-apm-server-ea]]
+// === Standalone Elastic Agent-managed APM Server
+// // I really don't know how to sell this option
+// Instead of installing and configuring the APM Server binary, let {agent} orchestrate it for you.
+// Install {agent} and manually configure the agent locally on the system where it's installed.
+// You are responsible for managing and upgrading {agent}. This approach is recommended for advanced users only.
+
+// **Pros**:
+
+// - Easily add integrations for other data sources
+// useful if EA already in place for other integrations, and customers want to customize setup rather than using Fleet for configuration
+// // TODO:
+// // maybe get some more hints on this one from the EA team to align with highlighting the same pros & cons.
+
+// **Available on Elastic Cloud**: ❌
+
+// This supports all of the same outputs as binary
+// see https://github.com/elastic/apm-server/issues/10467
+// **Supported outputs**:
+
+// **Configuration method**: YAML
+
+// image::./images/ea-ov.png[APM Server ea overview]
+
+// @simitt's notes for how to include EA-managed in the decision tree:
+// ****
+// If we generally describe Standalone Elastic Agent managed APM Server then we should also add it to this diagram:
+// Do you want to use other integrations?
+// -> yes: Would you like to use the comfort of Fleet UI based management? -> yes: Fleet managed APM Server; -> no: Standalone Elastic Agent managed APM Server
+// -> no: What is your prefered way of configuration? -> yaml: APM Server binary; -> Kibana UI: Fleet managed APM Server
+// ****
+
+// Components required:
+
+// [options="header"]
+// |====
+// | Installation method                         | APM Server | Elastic Agent | Fleet Server
+// | APM Server binary                           | ✔️          |               |
+// // | Standalone Elastic Agent-managed APM Server | ✔️          | ✔️             |
+// | Fleet-managed APM Server                    | ✔️          | ✔️             | ✔️
+// |====
+
+[float]
+== Help me decide
+
+Use the decision tree below to help determine which method of configuring and running the APM Server is best for your use case.
+
+[subs=attributes+]
+include::{observability-docs-root}/docs/en/observability/apm/diagrams/apm-decision-tree.asciidoc[APM Server decision tree]
+
+////
+Subsections
+////
+:leveloffset: +1
+include::{observability-docs-root}/docs/en/observability/apm/getting-started-apm-server/_apm_server_binary.asciidoc[]
+include::{observability-docs-root}/docs/en/observability/apm/getting-started-apm-server/get-started-with-fleet-apm-server.asciidoc[]
```

Comment:
I wonder if we can reverse the order here. I think we want to promote fleet-managed over the binary. @axw do you know the story here?

Reply:
<img width="290" alt="Screenshot 2024-09-10 at 3 27 52 PM" src="https://github.com/user-attachments/assets/fdc908cc-c19a-4b47-96a3-ca8b325fde0a">

Reply:
I went ahead and switched the order in the [Get started](https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-getting-started-apm-server.html) section . We can revert if there's a reason to lead with the binary instead.